.travis.yml

```diff
@@ -27,6 +23,8 @@ jobs:
         - npm install react-bootstrap --save
         - npm install react-horizontal-scrolling-menu --save
         - npm install axios --save
+        - npm install enzyme --save
```

Comment:
Your work with the Travis CI service & misc. documentation does not go unappreciated, thank you!

Reply:
No problem!